forge-gui/res/cardsfolder/upcoming/binding_negotiation.txt

```diff
@@ -0,0 +1,7 @@
+Name:Binding Negotiation
+ManaCost:1 B
+Types:Sorcery
+A:SP$ Discard | ValidTgts$ Opponent | Mode$ RevealYouChoose | DiscardValid$ Card.nonLand | DiscardValidDesc$ nonland card | RememberDiscarded$ True | SubAbility$ DBReturn | SpellDescription$ Target opponent reveals their hand. You may choose a nonland card from it. If you do, they discard it. Otherwise, you may put a face-up exiled card they own into their graveyard.
+SVar:DBReturn:DB$ ChangeZone | Origin$ Exile | Destination$ Graveyard | ChangeValid$ Card.faceUp+TargetedPlayerOwn | Optional$ True | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ DBCleanup
```

Comment:
are you sure this part works?

Reply:
Still not working, The prompt talks about "do you want to put binding negotiation into graveyard" so I think the changetype isn't correct ?

Reply:
think about the two main modes of ChangeZone and which one you'd be using here by default

Reply:
Do we need another choosecard line with changezoneall ?

Reply:
no, you need Hidden$

Reply:
Thanks working now,